src/reducers/playlistReducer.js

```diff
@@ -0,0 +1,18 @@
+const playlistReducer = (state, { type, payload }) => {
+  switch (type) {
+    case "INITIALISE_PLAYLISTS":
+      return [...payload.playlists];
+    case "ADD_PLAYLIST":
+      return [...payload.playlists];
+    case "DELETE_PLAYLIST":
+      return [...payload.playlists];
+    case "ADD_TO_PLAYLIST":
+      return [...payload.playlists];
+    case "REMOVE_FROM_PLAYLIST":
+      return [...payload.playlists];
+    default:
+      return state;
```

Comment:
What is this reduced doing?

Looks like you are modifying data in action, that's an antipattern. Reducers should have control.